src/Compute/Compute/ChangeLog.md

```diff
@@ -20,6 +20,7 @@
 
 -->
 ## Upcoming Release
+* Removed all `ContainerService` cmdlets. The Container Service API was deprecated in January 2020. 
```

Comment:
I suggest to list all removed cmdlets by name here, to make it easier for users to recognize what were removed.